src/pages/PomodoroPage/PomoSettingsModal/PomoSettingsModal.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { FC, useState, memo, useCallback, useMemo } from 'react';
+import { message } from 'antd';
+import Modal, { ModalProps } from 'components/Modal';
+import Button from 'components/Button';
+import {
+  defaultLongRestMinutes,
+  defaultRestMinutes,
+  pomoRestSettings,
+} from 'config/pomoconf';
+import { SettingsItem, StyledInputNumber } from './PomoSettingsModal.styles';
+
+enum RestType {
+  short = 'short',
+  long = 'long',
+}
+
+type RestMinutesType = Record<RestType, number>;
+
+const PomoSettingsModal: FC<ModalProps> = ({
+  isOpen,
+  title,
+  onClose,
+}: ModalProps) => {
+  const getRestSettings = useCallback(() => {
```

Comment:
для ясности мб лучше сделать так:
```suggestion
  const getRestSettings = useCallback((): RestMinutesType => {
```

ну и разрулить там JSON.parse ниже, если понадобится